testdata/serviceconfig.yml

```diff
@@ -30,7 +30,17 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
+        template_expressions:
```

Comment:
I like the flexibility this provides. I worry a bit about being able to grok the whole bit. As bad as it was to have common/combined hardcoded, it made it a *whole* lot simpler to understand from an operator's perspective (they didn't have to think about it).

Here, we give them half the information -- what the template vars are populated with. We don't declare what the actual template is.  We should think about a way of exposing that that is concise and easy to understand.

For comparison, here is the apache config:
```
LogFormat "%h %l %u %t \"%r\" %>s %b" common
CustomLog logs/access_log common
```

Interestingly, they provide the other half of the information (requiring knowledge of what the percent directives refer to).

I wonder if we should do the mapping in the definition of common and just let this aspect config consist entirely of `logName` and `logFormat`, as it does now.